render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/setup/IntensityCorrectionSetup.java

```diff
@@ -58,4 +61,52 @@ public void initDefaultValues() {
 
 		this.intensityAdjust.initDefaultValues();
 	}
+
+	/**
+	 * @param  baseDataUrl                            base web service URL for data.
+	 * @param  stackWithZValues                       identifies stack and z layers to align.
+	 *
+	 * @param  deriveMatchCollectionNamesFromProject  indicates whether derived match collection names
+	 *                                                should be derived from the stack's project
+	 *                                                (default is to derive from stack name).
+	 *
+	 * @param  matchSuffix                            suffix to append to derived match collection names
+	 *                                                (specify empty string to omit suffix).
+	 *                                                Suffix is needed when match aggregation is performed
+	 *                                                by an earlier pipeline step.
+	 * @return a clone of this setup populated with the specified parameters.
+	 */
+	public IntensityCorrectionSetup buildPipelineClone(final String baseDataUrl,
+													   final StackWithZValues stackWithZValues,
+													   final boolean deriveMatchCollectionNamesFromProject,
```

Comment:
I removed unused match parameters in a commit below.

Reply:
Thanks! I completely overlooked the mention of matches, here.